src/main/resources/export/cpp/PIDCommand.h

```diff
@@ -0,0 +1,37 @@
+#set($command = $helper.getByName($command-name, $robot))
+#macro( klass $cmd )#if( "#type($cmd)" == "" )frc::PIDCommand#else#type($cmd)#end#end
+#header()
+
+#ifndef #constant($command.name)_H
+\#define #constant($command.name)_H
+
+
+\#include "frc/commands/Subsystem.h"
+\#include "Robot.h"
+
+/**
+ *
+ *
+ * @author ExampleAuthor
+ */
+class #class($command.name): public frc::PIDCommand {
+public:
+#@autogenerated_code("constructor", "    ")
+#parse("${exporter-path}Command-constructor-header.h")
+#end
+
+    double returnPIDInput() override;
```

Comment:
You sure this compiles? Some of the functions marked override are spelled wrong.

Reply:
I prefer being specific (i.e., "include what you use"), and compilation times are measurably quicker that way. I think Brad has a different opinion on this though. Only one or two C++ examples use WPILib.h, but I forget which.